### PR TITLE
chore: update tekton tasks section

### PR DESCRIPTION
**Pull request by ksimon1**
chore: update tekton tasks section
this PR updates links and mentions about TTO (which is deprecated) and replace them with SSP operator

### CODE REVIEW

File: docs/virtual_machines/tekton_tasks.md [hunk 1]
```diff
@@ -125,9 +128,9 @@ KubeVirt-specific Tekton tasks, which are focused on:
...
 
-- [Windows customize](https://github.com/kubevirt/tekton-tasks-operator/blob/main/data/tekton-pipelines/okd/windows-customize-pipeline.yaml) - Pipeline will install sql server or vs code in windows VM. More informations about pipeline can be found [here](https://github.com/kubevirt/kubevirt-tekton-tasks/tree/main/examples/pipelines/windows-customize)
+- [Windows customize](https://github.com/kubevirt/ssp-operator/blob/main/data/tekton-pipelines/windows-customize-pipeline.yaml) - Pipeline will install sql server or vs code in windows VM. More informations about pipeline can be found [here](https://github.com/kubevirt/kubevirt-tekton-tasks/tree/main/examples/pipelines/windows-customize)
```
Comment (jcanocan) [inline docs/virtual_machines/tekton_tasks.md:136]:
```suggestion
- [Windows customize](https://github.com/kubevirt/ssp-operator/blob/main/data/tekton-pipelines/windows-customize-pipeline.yaml) - Pipeline will install SQL server or VS Code in a Windows VM. More informations about pipeline can be found [here](https://github.com/kubevirt/kubevirt-tekton-tasks/tree/main/examples/pipelines/windows-customize)
```

File: docs/virtual_machines/tekton_tasks.md [hunk 2]
```diff
@@ -1,24 +1,25 @@
...
+or it can be deployed by the user as a stand-alone from the latest [release](https://github.com/kubevirt/ssp-operator/releases/latest).
 
 !!! Note
     TTO requires [Tekton](https://tekton.dev/) to work.
 
 !!! Note
-    TTO does not deploy its resources by default.
+    SSP does not deploy KubeVirt Tekton tasks resources by default.
 
 The user has to enable `deployTektonTaskResources` feature gate in HCO CR to deploy all its resources
```
Comment (jcanocan) [inline docs/virtual_machines/tekton_tasks.md:24]:
```suggestion
The user has to enable `deployTektonTaskResources` feature gate in HCO CR to deploy all its resources:
```

File: docs/virtual_machines/tekton_tasks.md [hunk 3]
```diff
@@ -1,24 +1,25 @@
...
+[SSP](https://github.com/kubevirt/ssp-operator) is a Golang based operator, which takes care of deploying 
 [kubevirt-tekton-tasks](https://github.com/kubevirt/kubevirt-tekton-tasks) and example pipelines.
 
-TTO is shipped as a part of [hyperconverged-cluster-operator](https://github.com/kubevirt/hyperconverged-cluster-operator) 
-or it can be deployed by the user as a stand-alone from the latest [release](https://github.com/kubevirt/tekton-tasks-operator/releases/latest).
+SSP is shipped as a part of [hyperconverged-cluster-operator](https://github.com/kubevirt/hyperconverged-cluster-operator) 
+or it can be deployed by the user as a stand-alone from the latest [release](https://github.com/kubevirt/ssp-operator/releases/latest).
 
 !!! Note
     TTO requires [Tekton](https://tekton.dev/) to work.
```
Comment (jcanocan) [inline docs/virtual_machines/tekton_tasks.md:19]:
```suggestion
> **Note:** SSP requires [Tekton](https://tekton.dev/) to work.
```

File: docs/virtual_machines/tekton_tasks.md [hunk 4]
```diff
@@ -1,24 +1,25 @@
...
 
 !!! Note
-    TTO does not deploy its resources by default.
+    SSP does not deploy KubeVirt Tekton tasks resources by default.
```
Comment (jcanocan) [inline docs/virtual_machines/tekton_tasks.md:22]: Same as above.

File: docs/virtual_machines/tekton_tasks.md [hunk 5]
```diff
@@ -48,15 +49,14 @@ User can use this command to enable `deployTektonTaskResources` feature gate in
 ```console
 oc patch hco kubevirt-hyperconverged  --type=merge -p '{"spec":{"featureGates": {"deployTektonTaskResources": true}}}'
 ```
-or in TTO CR if TTO is deployed as a stand-alone without HCO
+or in SSP CR if SSP is deployed as a stand-alone without HCO
```
Comment (jcanocan) [inline docs/virtual_machines/tekton_tasks.md:52]:
```suggestion
or in SSP CR if SSP is deployed as a stand-alone without HCO:
```

Comment (jcanocan) [reply docs/virtual_machines/tekton_tasks.md:52]: Same in line https://github.com/kubevirt/user-guide/pull/700/files#diff-a77db169553e543ddbc385a4516c88ce5fd572e287ff4df352928f33eafba3ddR48: 
"User can use this command to enable `deployTektonTaskResources` feature gate in HCO CR", add semicolum.

File: docs/virtual_machines/tekton_tasks.md [hunk 6]
```diff
@@ -125,9 +128,9 @@ KubeVirt-specific Tekton tasks, which are focused on:
 All these tasks can be used for creating [pipelines](https://github.com/tektoncd/pipeline/blob/main/docs/pipelines.md).
 TTO is creating multiple example pipelines, e.g.:
```
Comment (jcanocan) [inline docs/virtual_machines/tekton_tasks.md:129]:
```suggestion
SSP is creating multiple example pipelines, e.g.:
```
I'm not sure about this one. Please ignore if it's not accurate.

File: docs/virtual_machines/tekton_tasks.md [hunk 7]
```diff
@@ -35,10 +36,10 @@ spec:
 
 or in TTO CR:
```
Comment (jcanocan) [inline docs/virtual_machines/tekton_tasks.md:37]:
```suggestion
or in SSP CR:
```

**Review by jcanocan — COMMENTED** [review]
Thanks for taking the time to update the documentation. Nice job! I've added a couple of comments. Hope you find them useful :)

**Comment by jcanocan** [comment]
Looks good to me! Thanks @ksimon1!

**Comment by ksimon1** [comment]
@dhiller, @phoracek can you please review this PR?

**Review by dhiller — COMMENTED** [review]
/approve

File: docs/virtual_machines/tekton_tasks.md [hunk 8]
```diff
@@ -1,26 +1,23 @@
...
+SSP is shipped as a part of [hyperconverged-cluster-operator](https://github.com/kubevirt/hyperconverged-cluster-operator) 
+or it can be deployed by the user as a stand-alone from the latest [release](https://github.com/kubevirt/ssp-operator/releases/latest).
 
-!!! Note
-    TTO requires [Tekton](https://tekton.dev/) to work.
+**Note:** SSP requires [Tekton](https://tekton.dev/) to work.
 
-!!! Note
-    TTO does not deploy its resources by default.
+**Note:** SSP does not deploy KubeVirt Tekton tasks resources by default.
```
Comment (phoracek) [inline docs/virtual_machines/tekton_tasks.md:18]:
```suggestion
SSP does not deploy KubeVirt Tekton tasks resources by default.
```

You say that it is not deployed by default and then you document how to opt-in. No need for a note.

File: docs/virtual_machines/tekton_tasks.md [hunk 9]
```diff
@@ -48,15 +45,14 @@ User can use this command to enable `deployTektonTaskResources` feature gate in
 ```console
 oc patch hco kubevirt-hyperconverged  --type=merge -p '{"spec":{"featureGates": {"deployTektonTaskResources": true}}}'
 ```
-or in TTO CR if TTO is deployed as a stand-alone without HCO
+or in SSP CR if SSP is deployed as a stand-alone without HCO:
```
Comment (phoracek) [inline docs/virtual_machines/tekton_tasks.md:48]:
```suggestion
or by applying a patch on an existing SSP CR:
```

File: docs/virtual_machines/tekton_tasks.md [hunk 10]
```diff
@@ -48,15 +45,14 @@ User can use this command to enable `deployTektonTaskResources` feature gate in
...
 ```console
-oc patch TektonTasks tektontasks  --type=merge -p '{"spec":{"featureGates": {"deployTektonTaskResources": true}}}'
+oc patch ssp ssp  --type=merge -p '{"spec":{"featureGates": {"deployTektonTaskResources": true}}}'
 ```
 
-Once `spec.featureGates.deployTektonTaskResources` is set to `true`, TTO will not delete any cluster 
-tasks or pipeline examples even if it is reverted back to false.
+Once `spec.featureGates.deployTektonTaskResources` is set to `true`, SSP will not delete any tasks or pipeline examples even if it is reverted back to false.
```
Comment (phoracek) [inline docs/virtual_machines/tekton_tasks.md:53]:
```suggestion
**Note:** Once `spec.featureGates.deployTektonTaskResources` is set to `true`, SSP will not delete any tasks or pipeline examples even if it is reverted back to false.
```

**Review by phoracek — CHANGES_REQUESTED** [review]

**Review by phoracek — APPROVED** [review]
/lgtm
/approve

**Pull request merged**